rp235x-hal-examples/README.md

```diff
@@ -56,44 +56,119 @@ https://github.com/rp-rs/rp-hal-boards/ for more details.
 <!-- GETTING STARTED -->
 ## Getting Started
 
-To build all the examples, first grab a copy of the source code:
+To build the examples, first grab a copy of the source code:
 
 ```console
 $ git clone https://github.com/rp-rs/rp-hal.git
 ```
 
-Then use `rustup` to grab the Rust Standard Library for the appropriate targets. There are two targets because the RP2350 has both an Arm mode and a RISC-V mode.
-
+Then use `rustup` to grab the Rust Standard Library for the appropriate targets.
+RP2350 has two possible targets: thumbv8m.main-none-eabihf for the ARM cpu, and
+riscv32imac-unknown-none-elf for the RISC-V cpu.
 ```console
 $ rustup target add thumbv8m.main-none-eabihf
 $ rustup target add riscv32imac-unknown-none-elf
 ```
 
-To build all the examples for Arm mode, run:
+**Note: all examples assume the current directory is <repo root>/rp235x-hal-examples.**
+```
+cd rp235x-hal-examples
+```
 
+The most basic method is to use `cargo build` with the `--bin` flag to specify the example you want to
+build. For example, to build the `blinky` example:
 ```console
-$ cargo build --target=thumbv8m.main-none-eabihf
-   Compiling rp235x-hal-examples v0.1.0 (/home/user/rp-hal/rp235x-hal-examples)
-    Finished `dev` profile [unoptimized + debuginfo] target(s) in 4.53s
 $ cargo build --bin blinky
-    Finished `dev` profile [unoptimized + debuginfo] target(s) in 0.06s
+   Compiling proc-macro2 v1.0.89
+   Compiling unicode-ident v1.0.13
+   Compiling syn v1.0.109
+...
+   Compiling pio-parser v0.2.2
+   Compiling rp235x-hal v0.2.0
+   Compiling pio-proc v0.2.2
+    Finished `dev` profile [unoptimized + debuginfo] target(s) in 14.97s
+```
+
+This builds the default target, which is the for the ARM and the elf file
```

Comment:
Typo: "which is the for the"?

Raspberry Pi call it "Arm mode" and "RISC-V mode". We should use those terms.

Reply:
I've tried to change all occurrences and also changed "elf" to "ELF" in the text.